runtime/j9vm/javanextvmi.cpp

```diff
@@ -507,6 +507,18 @@ JVM_VirtualThreadUnmount(JNIEnv* env, jobject vthread, jboolean hide, jboolean l
 	}
 }
 
+JNIEXPORT void JNICALL
```

Comment:
The signatures of `JVM_VirtualThreadMount()` and `JVM_VirtualThreadUnmount()` have changed; see https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/master/src/hotspot/share/include/jvm.h#L1152-L1156.